### PR TITLE
Added Linkedin under Social media section

### PR DESCRIPTION
**Pull request by zubairzia0**

### CODE REVIEW

**Comment by zubairzia0** [comment]
I see that there is an issue already open for this, going to close this

**Comment by jholdstock** [comment]
Hey @zubairzia0,

I don't see any other PR which is implementing this. #630 and #686 are different - they are updating the footer on every page but not `beginner-guide.md`.

I think this can be merged if you want to re-open it.

**Comment by zubairzia0** [comment]
gotcha, thanks for pointing that out.

File: docs/getting-started/beginner-guide.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ Telegram, Rocket.Chat and KakaoTalk are partially bridged.
 ### Social Media
 
 * [Reddit](https://www.reddit.com/r/decred)
-
 * [Twitter](https://twitter.com/decredproject) (account operated by small group of contributors)
 * [Forum](https://forum.decred.org/)
 * [Facebook](https://facebook.com/decredproject) 
+* [Linkedin](https://www.linkedin.com/company/decredproject/) (account operated by small group of contributors)
```
Comment (jholdstock) [inline docs/getting-started/beginner-guide.md]:
```suggestion
* [LinkedIn](https://www.linkedin.com/company/decredproject/) (account operated by small group of contributors)
```

**Pull request merged**